03-Binary-Search-Trees/01-Ordered-Collections-Of-Data.md

```diff
@@ -23,23 +23,6 @@ Students should be able to:
 - [Slide Deck](https://docs.google.com/presentation/d/1Fj0deIUswGZ3ooJMpgVUqPEaWHKTkQ1w2Ci-yf8v66M/edit#slide=id.p)
 - [BST Exercise](https://github.com/Ada-C16/tree-practice)
```

Comment:
Is it ok for us to link to the C16 project here?

Reply:
Will make a note to go back and change once the C17 project exists